packages/xo-server/src/api/xostor.mjs

```diff
@@ -0,0 +1,173 @@
+import { asyncEach } from '@vates/async-each'
+import { createLogger } from '@xen-orchestra/log'
+
+const ENUM_PROVISIONING = {
+  Thin: 'thin',
+  Thick: 'thick',
+}
+const LV_NAME = 'thin_device'
+const PROVISIONING = Object.values(ENUM_PROVISIONING)
+const VG_NAME = 'linstor_group'
+const _XOSTOR_DEPENDENCIES = ['xcp-ng-release-linstor', 'xcp-ng-linstor']
+const XOSTOR_DEPENDENCIES = _XOSTOR_DEPENDENCIES.join(',')
+
+const log = createLogger('xo:api:pool')
+
+function pluginCall(xapi, host, plugin, fnName, args) {
+  return xapi.call('host.call_plugin', host._xapiRef, plugin, fnName, args)
+}
+
+async function destroyVolumeGroup(xapi, host, force) {
+  log.info(`Trying to delete the ${VG_NAME} volume group.`, { hostId: host.id })
+  return pluginCall(xapi, host, 'lvm.py', 'destroy_volume_group', {
+    vg_name: VG_NAME,
+    force: String(force),
+  })
+}
+
+async function installOrUpdateDependencies(host, method = 'install') {
+  if (method !== 'install' && method !== 'update') {
+    throw new Error('Invalid method')
+  }
+
+  const xapi = this.getXapi(host)
+  log.info(`Trying to ${method} XOSTOR dependencies (${XOSTOR_DEPENDENCIES})`, { hostId: host.id })
+  for (const _package of _XOSTOR_DEPENDENCIES) {
+    await pluginCall(xapi, host, 'updater.py', method, {
+      packages: _package,
+    })
+  }
+}
+
+export function installDependencies({ host }) {
+  return installOrUpdateDependencies.call(this, host)
+}
+installDependencies.description = 'Install XOSTOR dependencies'
+installDependencies.permission = 'admin'
+installDependencies.params = {
+  hostId: { type: 'string' },
+}
+installDependencies.resolve = {
+  host: ['hostId', 'host', 'administrate'],
```

Comment:
You can have the same name for the original and injected param (in this case `host`).